.ko.yaml

```diff
@@ -0,0 +1,3 @@
+defaultBaseImage: gcr.io/distroless/static:nonroot
+baseImageOverrides:
```

Comment:
We don't need the `baseImageOverrides` if we don't want to change it. I also think we don't need the `.ko.yaml` at all for now and we could rely on the default image `ko` uses, isn't it ?